src/drive/web/modules/filelist/FileListHeader.jsx

```diff
@@ -2,7 +2,10 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import classNames from 'classnames'
 import { translate } from 'cozy-ui/react/I18n'
-import { ActionMenu, MenuItem } from 'cozy-ui/react'
+import {ActionMenu, Button, MenuItem, Icon} from 'cozy-ui/react'
+import {toggleThumbnailSize} from "../navigation/duck/actions";
+import iconList from 'drive/mobile/assets/icons/icon-list.svg'
+import iconListMin from 'drive/mobile/assets/icons/icon-list-min.svg'
```

Comment:
can you put the icons in : `drive/assets/icons/` (I know i give you the previous folder, but it was just an exemple, sorry ^^)